syntax = "proto3";

package gRPCCommunication;

message GrpcParameter {
  string name = 1;
  string type = 2;
  string value = 3;
}